addon/components/md-nav.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-nav';
+
+const { run: { next }, Component } = Ember;
+
+export default Component.extend({
+  classNames: ['md-nav'],
+  layout,
+  tagName: 'nav',
```

Comment:
FYI, side-nav overlay affects the entire window in the chrome acceptance test runner, and the side-nav does not appear.  may not be something worth addressing here